### PR TITLE
Add Ownable Standard + Rarible Royalties V2 Specific Methods

### PR DESCRIPTION
**Pull request by xklob**
Adds the OZ-Ownable standard. Also adds the Rarible v2 specific methods needed to view royalty info.

### CODE REVIEW

**Comment by adhocmaster** [comment]
Looks great. We can refactor it to keep the claimOwner in another contract.

**Comment by xklob** [comment]
> Looks great. We can refactor it to keep the claimOwner in another contract.

Good call, just did so.

**Review by moahammadalt — DISMISSED** [review]

File: packages/contracts/contracts/external/RoyaltiesV2.sol [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts-upgradeable/interfaces/IERC2981Upgradeable.sol";
+
+abstract contract LibRoyalties2981 {
+    uint96 ROYALTY_FEE;        // royalty fee in percentage basis points (between 0 and 10,000)
+    address ROYALTY_RECIPIENT; // address of the recipient of the royalty fee
+
+    bytes4 constant _INTERFACE_ID_ROYALTIES = 0xcad96cca;
```
Comment (priyabhishek) [inline packages/contracts/contracts/external/RoyaltiesV2.sol:11]: Noob questions..
Will this cost storage? Can it be just a function?
e.g.
bytes4 constant getInterfaceIdRoyalties() { return "0xcad96cca"; }

Comment (xklob) [reply packages/contracts/contracts/external/RoyaltiesV2.sol:11]: `constant` vars are inlined into the bytecode - so zero storage cost (aside from the extra gas for 4 additional bytes in the bytecode)

Comment (xklob) [reply packages/contracts/contracts/external/RoyaltiesV2.sol:11]: (your suggestion would be the same gas cost, as it still would have to store the string literal in the bytecode, too)

**Review by sharif2008 — APPROVED** [review]

**Pull request merged**